assets/js/components/HostDetails/HostDetails.test.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { screen } from '@testing-library/react';
+import { screen, fireEvent } from '@testing-library/react';
```

Comment:
Use `userEvent` instead of `fireEvent`. It is the recommended library.
You will need to setup the user, etc
Check other test files with this approach